android-sdk/src/main/java/com/mobilecoin/lib/BlockchainClient.java

```diff
@@ -21,7 +21,7 @@
     private static final String TAG = BlockchainClient.class.getName();
     private static final BigInteger DEFAULT_TX_FEE = BigInteger.valueOf(10000000000L);
     private final Duration minimumFeeCacheTTL;
-    private ConsensusCommon.LastBlockInfoResponse lastBlockInfo;
+    private volatile ConsensusCommon.LastBlockInfoResponse lastBlockInfo;
```

Comment:
Don't store this response thread locally if it should be accessible from multiple threads